packages/terra-consumer-layout/src/Layout.jsx

```diff
@@ -19,6 +19,13 @@ const propTypes = {
   */
   helpItems: PropTypes.array,
   /**
+   * A center justified logo in header for mobile.
+   */
+  mobileLogo: PropTypes.oneOfType([
```

Comment:
I think if possible I'd prefer to have a single place to set logo. Right now we're also setting it here: https://github.com/cerner/terra-consumer/blob/master/packages/terra-consumer-nav/src/Nav.jsx#L33.

Ideally it would be like this:
```js
logo: {
  small: {}
  large: {}
}
```

Reply:
The one you referenced above is for `terra-consumer-nav`, however, the one I'm adding here is an optional config for `terra-consumer-layout`. So technically speaking, they're for two separate packages. Unless we want to keep nav and layout strongly coupled, it might be better to keep these two logos separate. Also, the new feature added in this PR is sort of an one-off feature for a specific client, moving forward I think most consumers will choose to stay with terra's UI team's recommendation to not display the center adjusted logo on mobile.